msresist/expectation_maximization.py

```diff
@@ -26,46 +26,27 @@ def EM_clustering(data, info, ncl, seqDist=None, gmmIn=None):
 
     # Indices for looking up probabilities later.
     idxx = np.atleast_2d(np.arange(d.shape[0]))
-
-    # In case we have missing data, use SVD-EM to fill it for initialization
-    pc = PCA(d, ncomp=5, missing="fill-em", standardize=False, demean=False, normalize=False)
-
-    # Solve for the KMeans clustering for initialization
-    km = KMeans(ncl, tol=1e-9)
-    km.fit(pc._adjusted_data)
-
-    # Add a dummy variable for the sequence information
     d = np.hstack((d, idxx.T))
 
     for _ in range(10):
         if gmmIn is None:
             # Initialize model
             dists = list()
             for ii in range(ncl):
-                nDist = [NormalDistribution(1.0, 0.2, min_std=0.01) for _ in range(d.shape[1] - 1)]
+                nDist = [NormalDistribution(sp.norm.rvs(), 0.2) for _ in range(d.shape[1] - 1)]
```

Comment:
This change is fine. It was meant to provide a better initialization, but that's not guaranteed.